lib/features/support/cucumber_types.rb

```diff
@@ -4,3 +4,10 @@
   type:        String,
   transformer: ->(s) { s }
 )
+
+ParameterType(
+  name:        'int_array',
+  regexp:      /\d+(?:, ?\d+)+/,
+  type:        String,
+  transformer: ->(s) { s.split(',').map(&:strip) }
```

Comment:
Probably doesn't matter but this could also convert the codes to integers

```suggestion
  transformer: ->(s) { s.split(',').map(&:strip).map(&:to_i) }
```

Reply:
Don't think it makes a difference for this change as we just go and join it back together into a string, but it might make a difference in future if we use the `int_array` type for something else.